ui/components/ExportModal.js

```diff
@@ -218,6 +243,18 @@ const ExportModal = (props) => {
           {ExtensibleButton && <ExtensibleButton {...props} closed={closed} />}
         </Box>
       </DialogContent>
+      <DialogActions>
+        <CustomTextTooltip
+          backgroundColor="#3C494F"
+          placement="top"
+          interactive={true}
+          title={renderTooltipContent()}
```

Comment:
Can you use this function which already has a predefined regex to identify a link 

https://github.com/layer5labs/meshery-extensions/blob/64d2f06b58d7f4795501f57e54e03cef6770149d/meshmap/src/sections/MesheryDesignerComponent/PatternServiceFormWrapper/RJSFCustomComponents/utils.js#L56

Reply:
I missed, you raised this PR in Meshery :) however we do have same function in Meshery too I guess you updated that right?